pages/characters/[id].page.jsx

```diff
@@ -6,6 +6,14 @@ import ImageViewer from "../../components/core/ImageViewer";
 import { Text, Box } from "@mantine/core";
 
 function Character({ character }) {
+  const getBreadcrumbs = (path) => {
+    const pathNames = path.split("/");
+    pathNames[
+      pathNames.length - 1
+    ] = `${character.first_name} ${character.last_name}`;
```

Comment:
I guess in the future this should switch the order depending on the local 🤔

Reply:
yee ill replace it later